onnxruntime/core/session/abi_session_options.cc

```diff
@@ -141,8 +141,17 @@ ORT_API_STATUS_IMPL(OrtApis::SetSessionGraphOptimizationLevel, _In_ OrtSessionOp
 }
 
 ORT_API_STATUS_IMPL(OrtApis::SetIntraOpNumThreads, _Inout_ OrtSessionOptions* options, int intra_op_num_threads) {
+#ifdef _OPENMP
+  ORT_UNUSED_PARAMETER(options);
+  ORT_UNUSED_PARAMETER(intra_op_num_threads);
+  return OrtApis::CreateStatus(ORT_INVALID_ARGUMENT,
```

Comment:
I saw this warning when running onnxruntime_perf_test.exe no matter whether I add -x parameter or not.

Reply:
You should see this error only when you've built ORT with --use_openmp. So this is expected. Irrespective of the value of intra_op_num_threads being used we want to let users know that you don't want to use this API when openmp is enabled.